index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+            The device associated with the event.
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section data-dfn-for="HIDInputReportEvent">
+      <h2>
+        `HIDInputReportEvent` interface
+      </h2>
+      <pre class="idl">
+        [Exposed=Window, SecureContext]
+        interface HIDInputReportEvent : Event {
+            constructor(DOMString type, HIDInputReportEventInit eventInitDict);
+            [SameObject] readonly attribute HIDDevice device;
+            readonly attribute octet reportId;
+            readonly attribute DataView data;
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn>device</dfn> attribute
+        </dt>
+        <dd>
+          The {{HIDDevice}} instance that represents the device that sent the
+          input report.
+        </dd>
+        <dt>
+          <dfn>reportId</dfn> attribute
+        </dt>
+        <dd>
+          The one-byte identification prefix for this report, or 0 if the
+          device does not [=use report IDs=].
+        </dd>
+        <dt>
+          <dfn>data</dfn> attribute
+        </dt>
+        <dd>
+          A {{DataView}} containing the data from the input report, excluding
+          the {{HIDInputReportEvent/reportId}} byte if the device [=uses report
+          IDs=].
+        </dd>
+      </dl>
+      <section data-dfn-for="HIDInputReportEventInit">
+        <h3>
+          `HIDInputReportEventInit` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary HIDInputReportEventInit : EventInit {
+              required HIDDevice device;
+              required octet reportId;
+              required DataView data;
+          };
+        </pre>
+        <dl>
+          <dt>
+            <dfn>device</dfn> member
+          </dt>
+          <dd>
+            The device associated with the event.
+          </dd>
+          <dt>
+            <dfn>reportId</dfn> member
+          </dt>
+          <dd>
+            The report ID for the input report.
+          </dd>
+          <dt>
+            <dfn>data</dfn> member
+          </dt>
+          <dd>
+            The data for the input report.
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section data-dfn-for="HIDCollectionInfo">
+      <h2>
+        `HIDCollectionInfo` dictionary
+      </h2>
+      <pre class="idl">
+        dictionary HIDCollectionInfo {
+            unsigned short usagePage;
+            unsigned short usage;
+            octet type;
+            sequence&lt;HIDCollectionInfo&gt; children;
+            sequence&lt;HIDReportInfo&gt; inputReports;
+            sequence&lt;HIDReportInfo&gt; outputReports;
+            sequence&lt;HIDReportInfo&gt; featureReports;
         };
       </pre>
+      <p>
+        When a device is first connected to the system, the system MUST
+        retrieve the [=Report descriptor=] for each [=HID interface=] exposed
+        by the device. The <dfn>Report descriptor</dfn> is a [=byte sequence=]
```

Comment:
I capitalized it since the DCD for HID uses that convention:

> Words in bold with initial letter capitalized
indicate elements with special meaning
such as requests, descriptors, descriptor
sets, classes, or subclasses

I don't think there's any reason we need to use that convention here since the keyword is already indicated and the definition is linked.